test/ebaze_web/controllers/page_controller_test.exs

```diff
@@ -1,11 +1,29 @@
 defmodule EbazeWeb.PageControllerTest do
   use EbazeWeb.ConnCase
 
+  alias Ebaze.Auctions
+
   describe "get landing page " do
     test "GET /", %{conn: conn} do
       conn = get(conn, "/")
       assert html_response(conn, 200) =~ "Welcome to Ebaze!"
       assert html_response(conn, 200) =~ "New auction items listed everyday. Sell or Bid now!"
     end
+
+    test "GET / displays auction listings", %{conn: conn} do
```

Comment:
Q: in rails you can test that certain vars are assigned to the template....is that possible in exunit?

Reply:
Not sure if this will resolve it -  we can use `conn.assigns` to store value for key in conn and check for that in the response

Reply:
yeah, that could be good too